cmd/zt_copy_blob_upload_test.go

```diff
@@ -435,6 +435,68 @@ func (s *cmdIntegrationSuite) TestUploadDirectoryToContainerWithPattern(c *chk.C
 	})
 }
 
+func (s *cmdIntegrationSuite) TestUploadDirectoryToContainerWithIncludeBefore_UTC(c *chk.C) {
+	s.doTestUploadDirectoryToContainerWithIncludeBefore(true, c)
+}
+
+func (s *cmdIntegrationSuite) TestUploadDirectoryToContainerWithIncludeBefore_LocalTime(c *chk.C) {
+	s.doTestUploadDirectoryToContainerWithIncludeBefore(false, c)
+}
+
+func (s *cmdIntegrationSuite) doTestUploadDirectoryToContainerWithIncludeBefore(useUtc bool, c *chk.C) {
+	bsu := getBSU()
+
+	// set up the source with numerous files
+	srcDirPath := scenarioHelper{}.generateLocalDirectory(c)
+	defer os.RemoveAll(srcDirPath)
+	originalContent := scenarioHelper{}.generateCommonRemoteScenarioForLocal(c, srcDirPath, "")
+
+	// add newer files, which we wish to include
+	filesToInclude := []string{"important.txt", "includeSub/amazing.txt", "includeSub/wow/amazing.txt"}
+	scenarioHelper{}.generateLocalFilesFromList(c, srcDirPath, filesToInclude)
```

Comment:
in this case, the extra filesToInclude are not very meaningful, since they are just included along with the rest of the originalContent